### DISCUSSION TITLE
UWP support

### DISCUSSION THREAD

**Discussion by vladislav1nikov** [bevyengine/bevy#8162] [Q&A] [answered]
Does Bevy support UWP or only Windows desktop app?

**Comment by james7132** [comment, marked as answer]
Currently, we only support Win32 apps. UWP seems to be reliant on Microsoft's .NET stack, which Rust completely circumvents. This will require some more investigation at the minimum to support, but it definitely seems like a quick(-ish) venue to get the engine supporting Xbox as a platform.

**Reply by jdm** [reply]
UWP is unrelated to .NET. There's a tier3 rust target that can be used with nightly rustc versions, and then you need to ensure that none of your code or dependencies use forbidden win32 APIs. I shipped a UWP Rust app in the HoloLens store in 2020; it's definitely an undertaking but possible.

**Comment by momo-AUX1** [comment]
soon :)
